hw/hw2-baseball.sql

```diff
@@ -0,0 +1,67 @@
+-- BDA_696 HW2 Script
+-- Each table will open with all players
+-- and then again for just one player (110029)
+-- for verification
+
+USE baseball;
+
+DROP TABLE IF EXISTS career_avg,
+	season_avg,
+	rolling_avg;
+    
+-- Create career average table
+
+CREATE TABLE career_avg
+	SELECT batter,
+		SUM(Hit) / SUM(atBat) AS bat_avg_career
+	FROM batter_counts JOIN game ON batter_counts.game_id = game.game_id WHERE atBat > 0
```

Comment:
JOIN game ON batter_counts.game_id = game.game_id

This join doesn't seem necessary in this statement. Everything you are selecting is from the batter_counts table